examples/cra-kitchen-sink/src/stories/index.js

```diff
@@ -54,75 +54,31 @@ const InfoButton = () =>
     {' '}Show Info{' '}
   </span>;
 
+const withBorder = (storyFn, context, bcolor) =>
+  <div style={{ padding: 6, border: `${bcolor} solid 2px` }}>
+    {storyFn()}
+  </div>;
+
 storiesOf('Button', module)
-  .addDecorator(withKnobs)
+  .getAddons(addInfo, withNotes, withBorder)
+  .add('with addons', context =>
+    context
```

Comment:
This is not really an api I'm really enthusiastic about to be honest.

I don't like it because the interface is modified by addons. I fear that this is too flexible, and can lead us to all sorts of problems.